### PR TITLE
feat: new commands

### PR DESCRIPTION
**Pull request by RadNotRed**
resume, stop, currentlyplaying commands

### CODE REVIEW

File: src/commands/help.ts [hunk 1]
```diff
@@ -1,27 +1,91 @@
...
-                    "title": `muusik.app Commands`,
-                    "description": `Here are the commands you can use with the muusik.app bot:`,
-                    "color": colors.Muusik,
-                    "fields": fields,
-                },
+                    title: `muusik.app Commands`,
+                    description: `Here are the commands you can use with the muusik.app bot:`,
+                    color: colors.Muusik,
+                    fields: fields,
+                    footer: { text: `Page 1 of ${totalPages}` }
```
Comment (LumiFae) [inline src/commands/help.ts:41]: isn't this supposed to be dynamic? or did we forget to track the page number? lol

**Review by LumiFae — CHANGES_REQUESTED** [review]
left 1 comment

File: src/commands/help.ts (hunk 1 above)
Comment (RadNotRed) [reply src/commands/help.ts:41]: Page 1 is there because that's the first time you run the command, it's dynamic below when you press buttons

Comment (LumiFae) [reply src/commands/help.ts:41]: ignore me, im dumb :3

**Review by LumiFae — APPROVED** [review]

**Pull request merged**